docker-compose.makefile.m1.yml

```diff
@@ -0,0 +1,65 @@
+version: '2.4'
+services:
+  mysql:
+    container_name: mattermost-mysql
+    platform: "linux/x86_64"
+    extends:
+        file: ./docker-compose.makefile.yml
+        service: mysql
+  postgres:
+    container_name: mattermost-postgres
+    extends:
+        file: ./docker-compose.makefile.yml
+        service: postgres
+  minio:
+    container_name: mattermost-minio
+    extends:
+        file: ./docker-compose.makefile.yml
+        service: minio
+  inbucket:
+    container_name: mattermost-inbucket
+    extends:
+        file: ./docker-compose.makefile.yml
+        service: inbucket
+  openldap:
+    container_name: mattermost-openldap
+    extends:
+        file: ./docker-compose.makefile.yml
+        service: openldap
+  elasticsearch:
+    container_name: mattermost-elasticsearch
+    image: "docker.elastic.co/elasticsearch/elasticsearch:7.10.2"
```

Comment:
If my understanding is correct, this is the only difference between the `docker-compose.common.yml` right?

Reply:
this is one, the other is mysql needs to be run as x86_64 as they don't support arm64 for  the time being

Reply:
I don't think bumping up the required version to 7 is viable for this. We need to test the code against the minimum supported version, and it needs to work for all platforms. A developer running against 7 in darwin/arm64 will miss out on bugs if they don't test against the right version.

We should look into what are the errors we are getting with 6.5.1 and possibly fix them, or wait for the minimum required version to be bumped to 7.

Reply:
if there is a way I can help with that, I'm all in.

but as far as I saw, it was the supported version, when I tried the current one it just kept retrying to connect and no errors in the log

Reply:
taking a quick look, seems that [official docker](https://hub.docker.com/_/elasticsearch?tab=tags&page=1&ordering=last_updated) images with arm support start from v7.8.0

if someone creates a Mattermost image of a supported elastic search version with arm support, I can try that on my machine.